### PR TITLE
added linux distributions

### PR DESCRIPTION
**Pull request by aedler-ft**

### CODE REVIEW

File: product-service-description/compliance-as-a-service/produkte/platform-linux.md [hunk 1]
```diff
@@ -12,8 +12,33 @@ Other than that, no specific software needs to be installed (aside from the depe
...
-- Both Entity Framework and SQLite queues can be launched on Linux, starting from version 1.3.3. 
-- Currently, SCU support is limited to the fiskaly and the Cryptovision packages, starting from version 1.3.3. Other SCUs (like Swissbit) will be supported soon.
+### Local storage support
+
+|                                                              | MX Linux   | Manjaro    | Linux Mint | Ubuntu     | Debian     |
+| ------------------------------------------------------------ | ---------- | ---------- | ---------- | ---------- | ---------- |
+| local mySQL Storage                                          | not tested | not tested | not tested | not tested | not tested |
+| local [SQLite-Storage](../features/SQLite-Storage.md)        | not tested | not tested | not tested | not tested | not tested |
+| local [EF-Storage](../features/EF-Storage.md)                | not tested | not tested | not tested | not tested | not tested |
+| Cloud-based [Azure-table-Storage](../features/Azure-table-Storage.md) | not tested | not tested | not tested | not tested | not tested |
```
Comment (TSchmiedlechner) [inline product-service-description/compliance-as-a-service/produkte/platform-linux.md]: Let's remove this Azure Storage, it's not used by anyone and not available in production :)

File: product-service-description/compliance-as-a-service/produkte/platform-linux.md [hunk 2]
```diff
@@ -12,8 +12,33 @@ Other than that, no specific software needs to be installed (aside from the depe
 
 ## Supported packages
 
-- Both Entity Framework and SQLite queues can be launched on Linux, starting from version 1.3.3. 
-- Currently, SCU support is limited to the fiskaly and the Cryptovision packages, starting from version 1.3.3. Other SCUs (like Swissbit) will be supported soon.
+### Local storage support
+
+|                                                              | MX Linux   | Manjaro    | Linux Mint | Ubuntu     | Debian     |
+| ------------------------------------------------------------ | ---------- | ---------- | ---------- | ---------- | ---------- |
+| local mySQL Storage                                          | not tested | not tested | not tested | not tested | not tested |
```
Comment (TSchmiedlechner) [inline product-service-description/compliance-as-a-service/produkte/platform-linux.md]: I'd shrink this list down to OSs that we actually tested and go with the following list:
- Ubuntu (tested: MySQL, SQLite)
- Debian (tested: MySQL, SQLite)

MX Linux and Majara are IMO anyway not used in this context, Linux Mint is just another Debian derivation (like Ubuntu, just a different UI)

Comment (TSchmiedlechner) [reply product-service-description/compliance-as-a-service/produkte/platform-linux.md]: We could add a paragraph below or above that states that technically, it should work on all distributions that use Mono, but we haven't had the chance yet to do test this in detail.

File: product-service-description/compliance-as-a-service/produkte/platform-linux.md [hunk 3]
```diff
@@ -12,8 +12,33 @@ Other than that, no specific software needs to be installed (aside from the depe
...
+| Cloud-based [Azure-table-Storage](../features/Azure-table-Storage.md) | not tested | not tested | not tested | not tested | not tested |
+
+**not tested:** should work, but not systematically quality managed.
+
+Both Entity Framework and SQLite queues can be launched on Linux, starting from version 1.3.3. 
+
+### TSE support
+
+|                                                              | MX Linux   | Manjaro                    | Linux Mint | Ubuntu                                 | Debian     |
+| ------------------------------------------------------------ | ---------- | -------------------------- | ---------- | -------------------------------------- | ---------- |
```
Comment (TSchmiedlechner) [inline product-service-description/compliance-as-a-service/produkte/platform-linux.md]: Same as above, I'd just list Ubuntu and Debian here.

The following SCUs were tested on Ubuntu and Debian: Cryptovision, fiskaly, Swissbit Hardware

Others should work as you listed, but are not tested in detail yet

File: product-service-description/compliance-as-a-service/produkte/platform-linux.md (hunk 1 above)
Comment (aedler-ft) [reply product-service-description/compliance-as-a-service/produkte/platform-linux.md]: done

File: product-service-description/compliance-as-a-service/produkte/platform-linux.md (hunk 2 above)
Comment (aedler-ft) [reply product-service-description/compliance-as-a-service/produkte/platform-linux.md]: done

File: product-service-description/compliance-as-a-service/produkte/platform-linux.md (hunk 3 above)
Comment (aedler-ft) [reply product-service-description/compliance-as-a-service/produkte/platform-linux.md]: done

**Pull request merged**